migrations/2_deploy_contracts.js

```diff
@@ -11,6 +12,7 @@ const NitroAdjudicator = artifacts.require('NitroAdjudicator');
 
 module.exports = function(deployer) {
   deployer.deploy(Commitment);
+  deployer.deploy(Outcome);
```

Comment:
Is it actually necessary to deploy this `library`? Your test links the library to `TestOutcome` and then deploys the result.

Reply:
I think it is - when you're linking a library I _think_ you're linking to a version deployed on chain. In any case, I was following the pattern for `Commitment`.